tests/_core/test_properties.py

```diff
@@ -358,6 +359,17 @@ class TestMarker(ObjectPropertyBase):
     values = ["o", (5, 2, 0), MarkerStyle("^")]
     standardized_values = [MarkerStyle(x) for x in values]
 
+    def assert_equal(self, a, b):
+        a_path, b_path = a.get_path(), b.get_path()
+        assert_array_equal(a_path.vertices, b_path.vertices)
+        assert_array_equal(a_path.codes, b_path.codes)
+        assert a_path.simplify_threshold == b_path.simplify_threshold
+        assert a_path.should_simplify == b_path.should_simplify
+
+        assert a.get_joinstyle() == b.get_joinstyle()
+        assert a.get_transform().to_values() == b.get_transform().to_values()
+        assert a.get_fillstyle() == b.get_fillstyle()
+
```

Comment:
This test change was needed because here we need to compare `MarkerStyle` objects. This class does not define `__eq__` method so it is hard to compare its instances against each other. This was not needed before because the old `MarkerStyle` compatibility function returned the same matplotlib `MarkerStyle` object without creating a copy. This allowed the test to pass earlier becuase `marker1 == marker2` falls back to `marker1 is marker2` which was true when the object returned was not a copy but is no longer the case. Instead we now compare the attributes of the marker objects.